### DISCUSSION TITLE
Does Prisma have an extension mechanism to support other databases?

### DISCUSSION THREAD

**Discussion by Lce-Lemon** [prisma/prisma#24251] [Q&A]
### Question

Does Prisma have an extension mechanism to support other databases?

**Comment by janpio** [comment]
Yes and no.

If the database supported on of the SQL dialects we currently support (PostgreSQL, MySQL, CockroachDB, SQLite, SQL Server) you can create a driver adapter by looking at our own ones, and adapt that to the driver of the database.

If the database requires different SQL queries or even NoSQL commands, then unfortunately nothing "extension" like. You can of course fork both our repositories and try to add support yourself in the main code, but that is a _huge_ undertaking.

Are you looking for support for a specific database?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
